### PR TITLE
Xdist fix

### PR DESCRIPTION
**Pull request by ramriddick**

### CODE REVIEW

File: pytest_reportportal/plugin.py [hunk 1]
```diff
@@ -14,24 +16,48 @@
...
+            wait_launch(session.config.py_test_service.RP.rp_client)
 
-    PyTestService.start_launch(
-        session.config.option.rp_launch,
-        tags=session.config.getini('rp_launch_tags'),
-        description=session.config.option.rp_launch_description,
-    )
+
+def wait_launch(rp_client):
+    timeout = time.time() + 10
```
Comment (krasoffski) [inline pytest_reportportal/plugin.py]: I would avoid "magic" numbers and create file/module level const for `10`.

**Comment by krasoffski** [comment]
@ramriddick Could you please move timeout to const and I will merger PR.

File: pytest_reportportal/plugin.py (hunk 1 above)
Comment (ramriddick) [reply pytest_reportportal/plugin.py]: Yes you are right. It's done.
+ I changed the example of working with RPLogHandler in README

**Comment by ramriddick** [comment]
@krasoffski its done.

**Pull request merged**